actions/icon_overlay.rb

```diff
@@ -15,7 +15,9 @@ module SharedValues
 
     class IconOverlayAction < Action
       def self.run(params)
-        if not (ENV['ICON_OVERLAY_SOURCE_PATH'] || ENV['ICON_OVERLAY_ASSETS_BUNDLE'])
+        source_path = ENV['ICON_OVERLAY_SOURCE_PATH'] || ""
+        assets_path = ENV['ICON_OVERLAY_ASSETS_BUNDLE'] || ""
```

Comment:
`ENV['ICON_OVERLAY_ASSETS_BUNDLE'].to_s` will also give you `""` if the first half is `nil`, and more Ruby-y.

Reply:
But may be less obvious to non-Rubyists?